lib/axiom/fs/base/execute_context.js

```diff
@@ -94,8 +94,6 @@ export var ExecuteContext = function(fileSystem, stdio, path, args) {
    */
   this.callee = null;
 
-  /** @type {!Stdio} */
-  this.stdio = stdio;
```

Comment:
Even though this does not break anything right now, it would be better to leave this as a member. stdio is meant to be a container of any number of streams (see comment in `stdio.js`: Additional streams can be added as named properties.). The stdin, etc. members are just there for convenience shortcuts.

See my comment in `getNestedStdio` on why it would be better to leave stdio as a member.